quantization/image_classification/migraphx/resnet50/e2e_migraphx_resnet_example.py

```diff
@@ -0,0 +1,388 @@
+import os
+import onnx
+import glob
+import scipy.io
+import numpy as np
+import logging
+from PIL import Image
+import onnx
+import onnxruntime
+from onnxruntime.quantization import CalibrationDataReader, CalibrationMethod, create_calibrator, write_calibration_table
```

Comment:
Is `CalibrationMethod` used in this file?

Reply:
All I've done is leverage the existing TensorRT example here to mirror things for MIGraphX.

Reply:
done